apps/barista/src/pages/page-content.ts

```diff
@@ -35,4 +35,37 @@ export class BaPageContent {
   private _data = '';
 
   constructor(private _elementRef: ElementRef<HTMLElement>) {}
+
+  ngAfterContentInit(): void {
+    const allColorItems = document.querySelectorAll('.js-ba-copybutton');
```

Comment:
Is this also removed once we have the color components in place? Can we change the class name?